examples/jsm/helpers/OctreeHelper.js

```diff
@@ -0,0 +1,38 @@
+
+import { Group, Box3Helper } from 'three';
+
+class OctreeHelper {
+
+  constructor(octree = null, color = 0xffff00) {
+
+    if (!octree) return octree
+
+    this.subTrees = octree.subTrees;
+    this.color = color;
+
+    this.group = new Group();
+    this.createBoxes = this.traverseTree(this.subTrees, this.group);
+
+    return this.createBoxes;
+
+  }
+
+  traverseTree(currentTree, group) {
+
+    for (let i = 0; i < currentTree.length; i++) {
+
+      const box = new Box3Helper(currentTree[i].box, this.color);
```

Comment:
Since you create a `Box3Helper` per subtree, the helper will produce a lot of draw calls when visualizing a complex. I suggest to refactor the class like below which will end up in a single draw call no matter how complex the Octree is:
```js
class OctreeHelper extends LineSegments {

	constructor( octree, color = 0xffff00 ) {

		const vertices = [];

		function traverse( tree ) {

			for ( let i = 0; i < tree.length; i ++ ) {

				const min = tree[ i ].box.min;
				const max = tree[ i ].box.max;

				vertices.push( max.x, max.y, max.z ); vertices.push( min.x, max.y, max.z ); // 0, 1
				vertices.push( min.x, max.y, max.z ); vertices.push( min.x, min.y, max.z ); // 1, 2
				vertices.push( min.x, min.y, max.z ); vertices.push( max.x, min.y, max.z ); // 2, 3
				vertices.push( max.x, min.y, max.z ); vertices.push( max.x, max.y, max.z ); // 3, 0

				vertices.push( max.x, max.y, min.z ); vertices.push( min.x, max.y, min.z ); // 4, 5
				vertices.push( min.x, max.y, min.z ); vertices.push( min.x, min.y, min.z ); // 5, 6
				vertices.push( min.x, min.y, min.z ); vertices.push( max.x, min.y, min.z ); // 6, 7
				vertices.push( max.x, min.y, min.z ); vertices.push( max.x, max.y, min.z ); // 7, 4

				vertices.push( max.x, max.y, max.z ); vertices.push( max.x, max.y, min.z ); // 0, 4
				vertices.push( min.x, max.y, max.z ); vertices.push( min.x, max.y, min.z ); // 1, 5
				vertices.push( min.x, min.y, max.z ); vertices.push( min.x, min.y, min.z ); // 2, 6
				vertices.push( max.x, min.y, max.z ); vertices.push( max.x, min.y, min.z ); // 3, 7

				traverse( tree[ i ].subTrees );

			}

		}

		traverse( octree.subTrees );

		const geometry = new BufferGeometry();
		geometry.setAttribute( 'position', new Float32BufferAttribute( vertices, 3 ) );

		super( geometry, new LineBasicMaterial( { color: color, toneMapped: false } ) );

		this.octree = octree;
		this.color = color;

		this.type = 'OctreeHelper';

	}

}
```

Reply:
Good to know, thanks! added another commit.